e2e-tests/demand-backup-sharded/run

```diff
@@ -48,10 +48,9 @@ desc 'write data, read from all'
 run_mongos \
 	'db.createUser({user:"myApp",pwd:"myPass",roles:[{db:"myApp",role:"readWrite"}]})' \
 	"userAdmin:userAdmin123456@$cluster-mongos.$namespace"
-sleep 2
 run_mongos \
 	'sh.enableSharding("myApp","rs0")' \
-	"myApp:myPass@$cluster-mongos.$namespace"
+	"clusterAdmin:clusterAdmin123456@$cluster-mongos.$namespace"
```

Comment:
Why do we need this change?

Reply:
The error that we are seeing is that the test collection sometimes gets created on rs0, rs1 and rs2 randomly (mongo desides) and when we do backup we don't know on which one it was.
So with this we are pinning the database to rs0 replica so that we can check every time that it is in that exact location in the backup.
What I did before is I added this enableSharding but I didn't notice that the myApp user doesn't have permissions to actually run it, so I need to change it to clusterAdmin.